src/olympia/abuse/cinder.py

```diff
@@ -456,13 +456,29 @@ def flag_for_human_review(self, appeal=False):
             if appeal
             else NeedsHumanReview.REASON_ABUSE_ADDON_VIOLATION
         )
+        nhr_object = NeedsHumanReview(
+            version=self.version, reason=reason, is_active=True
+        )
         if self.version:
-            NeedsHumanReview.objects.get_or_create(
+            if not NeedsHumanReview.objects.filter(
```

Comment:
Nit: This double negative kind of reads wonky to me? Doesn't [get_or_create](https://docs.djangoproject.com/en/dev/ref/models/querysets/#get-or-create) return a boolean if the record was created?

Could you do something like 

```python
versions_to_log=[]
nhr_object, was_created = NeedsHumanReview.objects.get_or_create(...)

if was_created:
   versions_to_log.apend(self.versions)

...
```

Reply:
Is `not .... exists()` a _double_ negative?  🤔   How would you write it differently?

The PR changes it from `get_or_create` because we need to be able to pass `_no_automatic_activity_log=True` to `NeedsHumanReview.save`